packages/config/config/devices/0x018c/pm-240-zw-3button.json

```
@@ -0,0 +1,25 @@
+{
+  "manufacturer": "Dawon",
+  "manufacturerId": "0x018c",
+  "label": "PM-S240-ZW",
```

I think this is supposed to be
```suggestion
  "label": "PM-S340-ZW",
```
at least according to [this comment](https://github.com/zwave-js/node-zwave-js/issues/5132#issuecomment-1264405046) I apparently never followed up.

Can this be?

---

If that's the case, I suppose we should also add the other product types mentioned in https://github.com/zwave-js/node-zwave-js/issues/5132#issuecomment-1264369553

---

@AlCalzone  you right! 3 button type model is `PM-S340-ZW` 👍 [6b2846c](https://github.com/zwave-js/node-zwave-js/pull/5833/commits/6b2846c888fc1670d525f4b9ebc76d18fe59cf3b) fixed here